src/interactive.c

```diff
@@ -134,5 +134,6 @@ interactive_end(struct view *view)
 				view_snap_to_edge(view, "down");
 			}
 		}
+		cursor_set(&view->server->seat, "left_ptr");
```

Comment:
But `left_ptr` may not be the cursor we want, for example when `interactive_end()` ends with the cursor positioned over a text-widget.  Maybe material for another issue/pull request, but perhaps we could put a `FIXME` or `TODO` comment here?

Reply:
Also we could use `XCURSOR_DEFAULT` here too.

Reply:
When closing the menu we are using a `cursor_rebase(.., true)` which forces a cursor_exit + cursor_enter event which in turn should get the application to set a new cursor image. Maybe we could keep setting the cursor to `left_ptr` (or `XCURSOR_DEFAULT`) in case the cursor is not currently on a surface but additionally force a re-enter for the underlying surface (if there is one) so it has a chance to updates the cursor image.

Reply:
Sounds reasonable. I tried something like that, but couldn't convince myself it was working properly, and didn't pursue it further.

p.s. There is a similar incorrectly hard-coded `XCURSOR_DEFAULT` (and `TODO` comment) in `output_update_for_layout_change()` for what it's worth. I think I'm to blame for that one.

Reply:
> There is a similar incorrectly hard-coded XCURSOR_DEFAULT

I am not sure I understand the movement part of that one but I also didn't really look into the relation of the "global" seat cursor and output cursors yet.

Regarding the cursor image part of that: I think that one is actually fine but I guess you could throw an `cursor_update_focus(server, true)` in there as well. Just be aware that it may cause hard to diagnose bugs when used during touchpad scrolling, double click or similar cursor actions (see the comment in `cursor_rebase()`).